src/components/NotesList/NotesList.jsx

```diff
@@ -0,0 +1,28 @@
+import PropTypes from "prop-types";
+import Note from "./Note/Note";
```

Comment:
Can you please add an empty line before `Note` import
Let's add an `index.js file` inside Note folder and import/export Note component inside. After that we can write here just: `import Note from "./Note"` to avoid duplication.